class AuthFailedError(Exception):
    """Raised when authentication fails."""
    pass
